onyo/lib/items.py

```diff
@@ -185,9 +185,9 @@ def update_from_path(self,
 
         self._path = path
 
-        if self['onyo.is.asset'] and self.repo:
+        if self.repo and self.repo.is_asset_path(path):
```

Comment:
Is the change of `self.repo.is_asset_path(path)` rather than `self['onyo.is.asset']` due to templates being able to set pseudokeys?

Reply:
Yes. `OnyoRepo.get_asset_content()` expects an actual asset, which a template is not.
I suspect that distinction here will be removed soon, but I'm not sure yet.